src/ucs/stats/serialization.c

```diff
@@ -244,8 +250,30 @@ ucs_stats_serialize_binary_recurs(FILE *stream, ucs_stats_node_t *node,
     /* Name */
     ucs_stats_write_str(node->name, stream);
 
-    /* Counters */
-    ucs_stats_write_counters(node->counters, cls->num_counters, stream);
+    /* 
+       Fix warning: uninit_use_in_call: Using uninitialized element of 
+                    array "filtered_counters" when calling "ucs_stats_write_counters"
+    */
+    filtered_counters[0] = 0; 
+
+    filter_node = node->filter_node;
+    j = 0;   
+    for (i = 0; (i < node->cls->num_counters) && (i < 64); ++i) {
+        if (filter_node->counters_bitmask & UCS_BIT(i)) {
+            ucs_stats_counter_t counters_acc = 0;
```

Comment:
maybe just declare
 ucs_stats_counter_t filtered_counters[64] ={}
and then
  filtered_counters[j] += temp_node->counters[i];

Reply:
Fixed.